pages/about.html

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: About
+permalink: /about/
+weight: 4
+itemtype: "https://schema.org/AboutPage"
+---
+
+<div
+  id="about"
+  itemscope
+  itemtype="https://schema.org/Organization">
+  <h1>About</h1>
+  <div itemprop="description">
+    <p>We're a group of people interested in making high quality game randomizers, focusing in how easy it is to get started, choose your settings and setting up multiworld!</p>
+    <p>We're community members who create tools for the community, and with that we love to keep everything open source so that others can enjoy our work and build on top of what we have.</p>
+  </div>
```

Comment:
dont we have that in the readme to take from? :thinking:

Reply:
nope. I took this from opencollective